sklearn/metrics/_classification.py

```diff
@@ -1321,6 +1321,16 @@ def fbeta_score(
     Asymptotically, `beta -> +inf` considers only recall, and `beta -> 0`
     only precision.
 
+    The formula for F-beta score is:
+
+    .. math::
+
+       F_\\beta = \\frac{(1 + \\beta^2) \\text{tp}}
+                        {(1 + \\beta^2) \\text{tp} + \\text{fp} + \\beta^2 \\text{fn}}
```

Comment:
Very nit: The variables are introduced in the order: "tp", "fn" and "fp". I think it looks a little nicer to have the formula match the order:

```suggestion
                        {(1 + \\beta^2) \\text{tp} + \\beta^2 \\text{fn} + \\text{fp}}
```

Reply:
I want to match the formula given here: https://scikit-learn.org/dev/modules/model_evaluation.html#binary-classification, so I've changed the order in the description instead. Hope it's okay